src/components/SingleReview.jsx

```diff
@@ -1,32 +1,41 @@
 import { useState, useEffect } from "react";
-import { getSingleReview } from "./api";
+import { getSingleReview, patchVotes } from "./api";
 import { v4 as uuidv4 } from "uuid";
 import { useParams } from "react-router-dom";
 
-
 export default function SingleReview({ isLoading, setIsLoading }) {
+  const [votes, setVotes] = useState(0);
+  
 
-  const [review, setReview] = useState({
-    "reviews": {
-    }
-  });
+  const [review, setReview] = useState({ reviews: {} });
   const { review_id } = useParams();
 
- 
- 
 
- 
   useEffect(() => {
+    setIsLoading(true);
     getSingleReview(review_id).then((reviewFromApi) => {
-      setReview(reviewFromApi);
+      setReview(reviewFromApi)
+      
       setIsLoading(false);
     });
   }, [review_id]);
 
-  function upVote() {
-   return review.reviews.votes  += 1
+  useEffect(() => {setVotes(review.reviews.votes); },[review])
+    
+
+  
+  function upVote(review_id) {
+    console.log(votes);
+    setVotes((votes) => votes + 1)
+    patchVotes(review_id, votes);
```

Comment:
The increment votes bit doesnt persist after refreshing i.e. its not actually updating the database